src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -43,21 +43,43 @@ variables:
 - name: isPRTrigger
   value: ${{ eq(variables['Build.Reason'], 'PullRequest') }}
 
-stages:
-- ${{ if and(ne(variables.isPRTrigger, 'true'), eq(variables['System.TeamProject'], 'internal')) }}:
-  - template: templates/stages/vmr-scan.yml
+- template: /eng/common/templates-official/variables/pool-providers.yml@self
 
-- template: /src/installer/eng/pipelines/templates/stages/vmr-build.yml
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
   parameters:
-    isBuiltFromVmr: true
-    isSourceOnlyBuild: ${{ variables.isSourceOnlyBuild }}
-    ${{ if eq(variables.isScheduleTrigger, 'true') }}:
-      scope: full
-    ${{ elseif eq(variables.isSourceOnlyBuildLite, 'true') }}:
-      scope: lite
-    ${{ elseif and(eq(variables.isPRTrigger, 'true'), eq(variables.isSourceOnlyBuild, 'true')) }}:
-      scope: ultralite
-    ${{ elseif and(eq(variables.isPRTrigger, 'true'), ne(variables.isSourceOnlyBuild, 'true')) }}:
-      scope: lite
-    ${{ else }}:
-      scope: full
+    sdl:
+      sourceAnalysisPool:
+        name: $(DncEngInternalBuildPool)
+        image: 1es-windows-2022
+        os: windows
+
+    componentgovernance:
+      sourceScanPath: $(Agent.BuildDirectory)/vmr
+      ignoreDirectories: $(Agent.BuildDirectory)/vmr/src

```

Comment:
I don't see this snippet being used in your test build. It doesn't look to be valid. I tried using this and it fails to load the pipeline.

Reply:
Oh yeah, I might have added it too late. Too bad, this comes straight out of 1ES docs for the templates :-/
Will give it another go.

Reply:
I moved to the right pipelines